core/common-util/src/main/java/uk/gov/gchq/gaffer/commonutil/elementvisibilityutil/Authorisations.java

```diff
@@ -120,7 +119,7 @@ public String toString() {
             ArrayByteSequence auth = (ArrayByteSequence) var3.next();
             sb.append(sep);
             sep = ",";
-            sb.append(new String(auth.toArray(), StandardCharsets.UTF_8));
```

Comment:
Interesting that this was introduced by the last code analysis PR. Wonder why this wasn't the suggested method originally.

Reply:
It's possible that checkstyle could detect this with different rules, although it might be beyond it's capabilities. It's strange it was done this way in the first place with a static and non-static import of the same thing. Won't happen again though as PMD looks for this now.